benchmarks/memory/src/main/scala/strawman/collection/MemoryFootprint.scala

```diff
@@ -41,11 +41,13 @@ object MemoryFootprint extends App {
       "List"          -> benchmark(List.fill(_)(obj)),
       "LazyList"      -> benchmark(LazyList.fill(_)(obj)),
       "scala.HashSet" -> benchmark(n => scala.collection.immutable.HashSet((1 to n).map(_.toString): _*)),
-      "HashSet"       -> benchmark(n => collection.immutable.HashSet((1 to n).map(_.toString): _*)),
+      "HashSet"       -> benchmark(n => strawman.collection.immutable.HashSet((1 to n).map(_.toString): _*)),
       "scala.TreeSet" -> benchmark(n => scala.collection.immutable.TreeSet((1 to n).map(_.toString): _*)),
-      "TreeSet"       -> benchmark(n => collection.immutable.TreeSet((1 to n).map(_.toString): _*)),
+      "TreeSet"       -> benchmark(n => strawman.collection.immutable.TreeSet((1 to n).map(_.toString): _*)),
```

Comment:
This didn't compile in master. We should compiler benchmarks as part of PR validation to prevent these regressions in the future.